app/controllers/application.js

```diff
@@ -3,10 +3,10 @@ import Ember from 'ember';
 export default Ember.Controller.extend({
 	isExpanded: true,
 	isNotLogin: Ember.computed('this.currentPath', function() {
-	  	if (this.currentPath !== "login") {
-	  		return true
+	  	if (this.currentPath !== 'login') {
```

Comment:
To be fair, this could be simplified further. I was trying to restrain myself from tangents that would require testing, while appeasing linters. ;)